src/script/view_model/content/AppLockViewModel.test.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import ko from 'knockout';
+import {Config} from '../../auth/config';
+import {ClientRepository} from '../../client/ClientRepository';
+import {ClientService} from '../../client/ClientService';
+import {User} from '../../entity/User';
+import {APPLOCK_STATE, AppLockViewModel} from './AppLockViewModel';
+
+type Writable<T> = {
```

Comment:
What about moving this to the [`TypeUtil` in commons](https://github.com/wireapp/wire-web-packages/blob/master/packages/commons/src/main/util/TypeUtil.ts) ?